CHANGELOG.md

```diff
@@ -44,18 +48,17 @@ See also the [GitHub releases page](https://github.com/FriendsOfSymfony/FOSHttpC
 
 ### Symfony HttpCache
 
-* BC BREAK: Renamed all event listeners to XxListener instead of XxSubscriber.
-* BC BREAK: Constructors for PurgeListener and RefreshListener now use an
+* **BC break:** Renamed all event listeners to XxListener instead of XxSubscriber.
+* **BC break:** Constructors for PurgeListener and RefreshListener now use an
   options array for customization.
 * Provide a trait for the event dispatching kernel, instead of a base class.
```

Comment:
this is actually a BC break as well, we removed the kernel to extend and only provide a trait now.